cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1330,6 +1331,46 @@ func waitForDevice(path string) error {
 	return nil
 }
 
+func candidateLabel(label string) (string, error) {
```

Comment:
this is assuming that label itself is lowercase, right? maybe merits a comment. I also wonder if the whole function shouldn't be a function exported by osutil/disks perhaps?